src/components/ListSkeleton.tsx

```diff
@@ -0,0 +1,26 @@
+import React, { FC, useRef } from 'react';
+import { Skeleton } from '@material-ui/lab';
+import { getRandomWidthPercent } from '../utils/skeleton-helpers';
+
+interface ListSkeletonProps {
+  arrayLength?: number;
+  height?: number;
+  minWidth?: number;
+  maxWidth?: number;
+}
+
+const ListSkeleton: FC<ListSkeletonProps> = ({ arrayLength, height, minWidth, maxWidth }) => {
+  const widthsRef = useRef(
+    Array.from({ length: arrayLength ?? 3 }, () => getRandomWidthPercent(minWidth ?? 10, maxWidth ?? 100))
+  );
+
+  return (
+    <>
+      {widthsRef.current.map((width, index) => (
+        <Skeleton key={index} width={width} height={height ?? 40} />
+      ))}
+    </>
+  );
+};
+
+export default ListSkeleton;
```

Comment:
sorry for at jeg sletta fila di `UserListSkeleton` nesten med en gang etter du hadde laget den, @elkebab, men tenkte det var nyttigere å ha en generell komponent som også kan brukes andre plasser der vi har lister. Synes du det er ok?

Reply:
Lurt!